pkg/auth/api/secrets/cleanup_test.go

```diff
@@ -135,58 +141,60 @@ func TestCleanupDeprecatedSecretsKnownConfig(t *testing.T) {
 	})
 	assert.NoError(t, err)
 
-	s, err := secrets.GetNamespaced(common.SecretsNamespace, secretName, metav1.GetOptions{})
+	s, err := secrets.Get(common.SecretsNamespace, secretName, metav1.GetOptions{})
 	assert.NoErrorf(t, err, "expected to find the secret %s belonging to the disabled auth provider", secretName)
 
 	err = CleanupClientSecrets(secrets, config)
 	assert.NoError(t, err)
 
-	s, err = secrets.GetNamespaced(common.SecretsNamespace, secretName, metav1.GetOptions{})
+	s, err = secrets.Get(common.SecretsNamespace, secretName, metav1.GetOptions{})
 	assert.Errorf(t, err, "expected to not find the secret %s belonging to the disabled auth provider", secretName)
 	assert.Nil(t, s, "expected the secret to be nil")
 }
 
-func getSecretInterfaceMock(store map[string]*corev1.Secret) v1.SecretInterface {
-	secretInterfaceMock := &fakes.SecretInterfaceMock{}
+func getSecretInterfaceMock(ctrl *gomock.Controller, store map[string]*corev1.Secret) wcorev1.SecretController {
```

Comment:
move from `getSecretInterfaceMock` to `getSecretControllerMock`?

Reply:
sounds reasonable, I'll make that tweak

Reply:
Looks like it didn't get changed to `getSecretControllerMock`?